pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -199,14 +205,31 @@ func (o *capability) loop() {
 		select {
 		case <-ctx.Done():
 			return
-		case r := <-o.newExpiryWorkerCh:
+		case r := <-o.newTimerCh:
 			o.wg.Add(1)
-			go o.expiryWorker(ctx, r)
+			go o.expiryTimer(ctx, r)
+		case resp := <-o.transmitCh:
+			o.handleTransmitMsg(ctx, resp)
 		}
 	}
 }
 
-func (o *capability) expiryWorker(ctx context.Context, r *request) {
+func (o *capability) handleTransmitMsg(ctx context.Context, resp *response) {
+	req, wasPresent := o.store.evict(ctx, resp.WorkflowExecutionID)
+	if !wasPresent {
```

Comment:
The debug log that was here can still be useful, I just thought that the message should be different if this is something expected. Or maybe no log for now is better.